sdk/identity/azure-identity/CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 
 ## 1.4.0b3 (Unreleased)
 
+- Preview of `VSCodeCredential` #10840
```

Comment:
This can be more helpful: what does `VSCodeCredential` do? How do I use it? Also, "#10840" can link to something--I suggest #10472 as a better target.